.github/workflows/grept-cronjob.yml

```diff
@@ -0,0 +1,135 @@
+          git checkout -b "$BRANCH"
+          git add .
+          git commit -m "fix: grept apply"
+          git push --set-upstream origin "$BRANCH"
+
+      - name: create PR body
+        if: steps.changes.outputs.detected == 'true'
+        id: prbody
+        run: |
+          tee prbody.md <<EOF
+          ## Repository governance update
+
+          This PR was automatically created by the AVM Team hive-mind using the [grept](https://github.com/Azure/grept) governance tool.
+
+          We have detected that some files need updating to meet the AVM governance standards.
+          Please review and merge with alacrity.
+
+          Grept config source: \`${{ env.GREPT_CONFIG }}\`
+
+          Thanks! The AVM team :heart:
+          EOF
+
+      - name: show body
+        if: steps.changes.outputs.detected == 'true'
+        run: |
+          echo "Displaying PR body:"
+          cat prbody.md
+
+      - name: create pull request
+        if: steps.changes.outputs.detected == 'true'
+        id: pr
+        run: |
+          PR_URL=$(gh pr create --title "chore: repository governance" --body-file prbody.md)
+          echo pull-request-number=$(gh pr view $PR_URL --json number | jq -r '.number') >> "$GITHUB_OUTPUT"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: close and comment out of date prs
+        if: steps.changes.outputs.detected == 'true'
+        run: |
+          PULL_REQUESTS=$(gh pr list --search "chore: repository governance" --json number,headRefName)
+          echo "$PULL_REQUESTS" | jq -r '.[] | select(.number != ${{ steps.pr.outputs.pull-request-number }}) | .number' | xargs -I {} gh pr close {} --delete-branch --comment "Supersceeded by #${{ steps.pr.outputs.pull-request-number }}"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: set env result=failure
+        if: ${{ failure() }}
+        run: |
+          echo 'result=failed' >> "$GITHUB_ENV"
+          if [ ! -z "${{ env.branch }}" ]; then
+            git push origin --delete "${{ env.branch }}"
+          fi
+
+
+      - name: set output
+        if: ${{ always() }}
+        id: set-output
+        run: |
+          echo "result=${{ env.result }}" >> "$GITHUB_OUTPUT"
+
+  report:
+    name: report
+    runs-on: ubuntu-latest
+    needs: governance
+    if: ${{ failure() }}
+    steps:
+      - name: raise issue
+        run: |
+          ## BLOCKED on matrix outputs: https://github.com/actions/runner/pull/2477
+          ## gh issue create --repo ${{ github.repository }} --title "Repository governance failure" --body "The following repositories failed governance checks:\n\n```json\n${{ toJSON(join(needs.governance.outputs)) }}\n```\n"
+          gh issue create --repo ${{ github.repository }} --assignee matt-FFFFFF --title "Repository governance failure" --body "This repository failed governance checks: <${{ github.server_url}}/${{ github.repository }}/actions/runs/${{ github.run_id}}>"
```

Comment:
Should this issue be created on the module repo or grept repo

Reply:
The issue would be created on module repo this time.